app/src/main/java/org/mozilla/fenix/components/metrics/LeanplumMetricsService.kt

```diff
@@ -65,12 +66,17 @@ class LeanplumMetricsService(private val application: Application) : MetricsServ
         LeanplumActivityHelper.enableLifecycleCallbacks(application)
 
         val installedApps = MozillaProductDetector.getInstalledMozillaProducts(application)
+        val backgroundServices = application.applicationContext.components.backgroundServices
```

Comment:
This may have performance implications. We might want to cache these items in the sharedPrefs and just pull them out at launch